src/rgw/rgw_civetweb.cc

```diff
@@ -120,6 +120,10 @@ void RGWCivetWeb::init_env(CephContext *cct)
     env.set("REMOTE_USER", info->remote_user);
   }
 
+  if (info->remote_addr) {
```

Comment:
@mikulely Agreed with your point. But I don't think there is a possibility of `if (info->remote_addr)` would take a false branch. Check `struct mg_request_info{};`.